cost/google/cud_recommendations/google_committed_use_discount_recommendations.pt

```diff
@@ -98,7 +115,7 @@ datasource "ds_google_project" do
   end
 end
 
-#https://cloud.google.com/compute/docs/reference/rest/v1/regions/list
+#GET LIST OF REGIONS
```

Comment:
Add white space between `#` and `GET`.

Reply:
Thanks @juanestrahcl , this was addressed.